lib/ohai/plugins/elixir.rb

```diff
@@ -15,7 +15,7 @@
 
 Ohai.plugin(:Elixir) do
   provides "languages/elixir"
```

Comment:
can probably freeze this one too, no?

Reply:
That wasn't showing up in the retained objects. The langs string was in there 10 times